src/util/removeEmpty.spec.js

```diff
@@ -0,0 +1,25 @@
+import assert from 'assert';
+import removeEmpty from './removeEmpty';
+
+describe('removeEmpty', () => {
+    context('with no empty values', () => {
+        it('should not remove any properties', () => {
+            const obj = { foo: 'fooval', bar: 'barval' };
+            assert.deepEqual(removeEmpty(Object.assign({}, obj)), obj);
```

Comment:
Thank you for your comment. I fixed it like `assert.deepEqual(removeEmpty({ ...obj }), obj);`.